src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchForm"
+      data-qa="big"
+      method="post"
+      action="#"
+    >
+      <div class="searchBar">
+        <label for="bigBar">
```

Comment:
do you need label?

Reply:
I thought the user would intuitively press the button to start typing.